Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM golang:1.20.7 as builder
 
-ARG CONFIG_TESTNET_TAG=T1.6.6.0
-ARG CONFIG_DEVNET_TAG=D1.6.6.1
-ARG CONFIG_MAINNET_TAG=v1.5.15.0
-ARG PROXY_TESTNET_TAG=v1.1.42
-ARG PROXY_DEVNET_TAG=v1.1.39
-ARG PROXY_MAINNET_TAG=v1.1.39
+ARG CONFIG_TESTNET_TAG=T1.6.10.0
+ARG CONFIG_DEVNET_TAG=D1.6.7.0
```

Comment:
won't we create a devnet release with latest?

Reply:
Will be referenced in the following days :pray: